marked-element.html

```diff
@@ -92,8 +92,25 @@
         observer: 'render',
         type: String,
         value: null
+      },
+      /** Conform to obscure parts of markdown.pl as much as possible. Don't fix any of the original markdown bugs or poor behavior. */
+      pedantic: {
+        observer: 'render',
+        type: Boolean,
+        value: false
+      },
+      /** Sanitize the output. Ignore any HTML that has been input. */
+      sanitize: {
+        observer: 'render',
+        type: Boolean,
+        value: false
+      },
+      /** Use "smart" typograhic punctuation for things like quotes and dashes. */
```

Comment:
nit: typo. should be typographic. Also, would you mind reformatting all these property comments to match the style we use in the other elements? So something like this: https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L77